airbyte-ci/connectors/metadata_service/lib/metadata_service/docker_hub.py

```diff
@@ -0,0 +1,61 @@
+import os
+import requests
+from typing import List
+
+
+def authenticate_docker_hub() -> str:
+    docker_username = os.environ.get("DOCKER_HUB_USERNAME")
+    docker_password = os.environ.get("DOCKER_HUB_PASSWORD")
+
+    if not (docker_username and docker_password):
+        raise ValueError("Please set the DOCKER_HUB_USERNAME and DOCKER_HUB_PASSWORD environment variables.")
+
+    auth_url = "https://hub.docker.com/v2/users/login/"
+    auth_data = {"username": docker_username, "password": docker_password}
+    response = requests.post(auth_url, json=auth_data)
+
+    if response.status_code != 200:
+        raise ValueError("Failed to authenticate with Docker Hub. Please check your credentials.")
+
+    token = response.json().get("token")
+    return token
+
+
+def get_image_tags(repo: str, image: str) -> List[str]:
+    token = authenticate_docker_hub()
```

Comment:
nit: the name of the method doesn't indicate to me that it's going to return a token for use

Reply:
Good call, will change.